packages/marko-web-search/browser/sort-select.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <select :class="className" @change="onChange">
```

Comment:
Change to `<select class="custom-select" @change="onChange">` and then remove the `className` from the props list.